src/lib/OpenEXR/ImfMisc.h

```diff
@@ -442,7 +442,7 @@ bool usesLongNames (const Header &header);
 //
 
 IMF_EXPORT
-int getChunkOffsetTableSize(const Header& header,bool deprecated_attribute=false);
+int getChunkOffsetTableSize(const Header& header);
```

Comment:
Should the comment about the deprecated_attribute be removed (line 440) as well as the parameter itself?

Reply:
Done, thanks.